website/source/docs/builders/googlecompute.html.md

```diff
@@ -104,6 +104,11 @@ location found:
 4.  On Google Compute Engine and Google App Engine Managed VMs, it fetches
     credentials from the metadata server. (Needs a correct VM authentication
     scope configuration, see above.)
+    
+ ### Alternative Authentication Method 
+   On your computer, if you have made your identity available as the 
+   Application Default Credentials by running gcloud auth application-default login, 
+   the provider will use your identity.
```

Comment:
Actually, `gcloud auth application-default login` creates the `application_default_credentials.json` in `%APPDATA%/gcloud/` // `$HOME/.config/gcloud/` which is described in step 3 of the "Precedence of Authentication Methods" chapter.

So may be this PR should better bring precision to step 3.

it could be something like:

```
 3.  A JSON file in a location known to the `gcloud` command-line tool.
     (`gcloud auth application-default login` creates it)	
```

Reply:
i made the change ! I'am agree with you